utils/display-tokens.i18n.js

```diff
@@ -1,74 +1,74 @@
 const i18n = {
-  "en": {
-    "color": {
-      "headers": {
+  en: {
+    color: {
+      headers: {
         name: "Colour name",
         token: "Token name",
         contrast: "Contrast",
         hex: "Hex",
-        rgb: "RGB"
+        rgb: "RGB",
       },
-      "body": {
+      body: {
         blue: "Blue",
         grayscale: "Grayscale",
         red: "Red",
         green: "Green",
-        yellow: "Yellow"
+        yellow: "Yellow",
       },
     },
-    "spacing": {
-      "headers": {
+    spacing: {
+      headers: {
         token: "Token name",
         px: "Pixel",
-        rem: "REM"
+        rem: "REM",
       },
-      "body": {},
+      body: {},
     },
-    "font": {
-      "headers": {
+    font: {
+      headers: {
         token: "Token name",
-        value: "Value (rounded)"
+        value: "Value (rounded)",
       },
-      "body": {
+      body: {
         heading: "Heading",
-        text: "Paragraph",
+        text: "Text",
         "text-long": "Paragraph",
         caption: "Caption",
         label: "Label",
       },
     },
-    "fontWeights": {
-      "headers": {
+    fontWeights: {
+      headers: {
         token: "Token name",
-        value: "Value"
+        value: "Value",
       },
-      "body": {
+      body: {
         light: "Light",
         regular: "Regular",
         medium: "Medium",
         semibold: "Semibold",
-        bold: "Bold"
+        bold: "Bold",
       },
     },
-    "fontFamilies": {
-      "headers": {
+    fontFamilies: {
+      headers: {
         token: "Token name",
-        value: "Value"
+        value: "Value",
       },
-      "body": {
+      body: {
         heading: "Heading",
         body: "Body",
         monospace: "Monospace",
-        icons: "Icons",
+        icons: "<gcds-icon name='tree'></gcds-icon>",
```

Comment:
We should keep this as "Icons".

Reply:
I left a comment in Figma too, I think we should remove this from the table? We don't recommend using that font style, rather, we provide them with an icon component!